rlaopt/atoms/affine.py

```diff
@@ -1,40 +1,121 @@
+"""Affine expression atom for optimization."""
+
 from __future__ import annotations
 
 import cvxpy as cp
 import torch
 
 from rlaopt.atoms.atom_expression import AtomExpression
-from rlaopt.expression.expression import Variable
+from rlaopt.expression import Expression, Variable
 
 
 class Affine(AtomExpression):
-    def __init__(self, x: Variable, A: torch.Tensor, b: torch.Tensor):
+    """Affine transformation atom: A @ x + b.
+
+    Represents an affine mapping of a variable or output of an expression.
+
+    Args:
+        x: Variable or Expression to transform.
+        A: Transformation matrix.
+        b: Bias/offset vector.
+
+    Raises:
+        TypeError: If x is not a Variable.
```

Comment:
Duplicate docs with the __init__. Delete in one place and update this with the new behaviors